tests/TestCase/I18n/TimeTest.php

```diff
@@ -507,6 +603,33 @@ public function testI18nFormatWithOffsetTimezone($class)
     }
 
     /**
+     * test formatting dates with offset style timezone and defaultOutputTimezone
+     *
+     * @dataProvider classNameProvider
+     * @see https://github.com/facebook/hhvm/issues/3637
```

Comment:
I think we never agreed on a certain order of those annotations, but personally I try to keep them in a kind of logical order.

1. parameter(s)
2. return value
3. throws
4. rest

For tests it's certainliy arguable whether ``@dataProvider`` can be seen as a parameter.
But ``@see`` clearly belongs to "the rest".